docs/install/index.md

```diff
@@ -1,9 +1,10 @@
 # Installation
 We use MLCommons CM Automation framework to run MLPerf inference benchmarks.
 
-Please follow the [official installation page](https://docs.mlcommons.org/ck/install) to install CM
+Please follow the below steps
 
-## Download the CM MLOps Repository
+1. [official installation page](https://docs.mlcommons.org/ck/install) prior to installing CM
```

Comment:
Thank you for this improvement. Here "to install CM" is appropriate than "prior to installing CM" as the link is for installing CM on different systems with all the needed dependencies. Can you please change this?
Also o can be in caps for Official

Reply:
Thank you Arjun. I updated for further clarity and your feedback. I might retract this PR and raise a issue for enhancement, pending CLA. Feel free to take over this change and resolve this.

Reply:
Thanks a lot @PurushGupta . Once CLA is signed, this PR can be merged.